C_CPP/Templates/readme.md

```diff
@@ -0,0 +1,18 @@
+# TEMPLATES
+
+<div align="center">
+<h2> What is a template in C++ programming?</h2>
+
+## 1. A template is believed to escalate the potential of C++ several fold by giving it the ability to define data types as parameters making it useful to reduce repetitions of the same declaration of classes for different data types. Declaring classes for every other data type(which if counted is way too much) in the very first place violates the DRY( Don’t Repeat Yourself) rule of programming and on the other doesn't completely utilise the potential of C++.
```

Comment:
keep a difference between heading and the body content.
make them left-align instead of center align.
keep only heading as bold and leave the body as normal texts.

Reply:
I have made changes.